x-pack/legacy/plugins/maps/public/angular/map_controller.js

```diff
@@ -332,7 +332,7 @@ app.controller('GisMapController', ($scope, $route, kbnUrl, localStorage, AppSta
   timefilter.disableAutoRefreshSelector();
   $scope.showDatePicker = true; // used by query-bar directive to enable timepikcer in query bar
   $scope.topNavMenu = [{
-    key: i18n.translate('xpack.maps.mapController.fullScreenButtonLabel', {
+    id: i18n.translate('xpack.maps.mapController.fullScreenButtonLabel', {
```

Comment:
More translations on id's

Reply:
The reason for this is that historically ID can be used as a label.
I aligned it to have an id and a translated label. Thanks for the catch!